keps/sig-api-machinery/4355-coordinated-leader-election/README.md

```diff
@@ -432,10 +413,13 @@ type CoordinatedLeaseStrategy string
 const (
   OldestCompatibilityVersionStrategy CoordinatedStrategy = "OldestCompatibilityVersionStrategy"
   NewestCompatibilityVersionStrategy CoordinatedStrategy = "NewestCompatibilityVersionStrategy"
+  NoCoordinationStrategy CoordinatedStrategy = "NoCoordinationStrategy"
```

Comment:
nit: the field is called `strategy`. No need to repeat that word in the value, i.e. `NoCoordination` is enough.

Reply:
Fixed.